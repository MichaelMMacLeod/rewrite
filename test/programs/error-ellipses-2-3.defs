(def (flatten (list (list $x ..) ..))
  (list $x .. .. ..))