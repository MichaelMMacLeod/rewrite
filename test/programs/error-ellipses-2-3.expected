error[E002]: too many ellipses
./test/programs/error-ellipses-2-3.defs:1:27
   |
 1 | (def (flatten (list (list $x ..) ..))
   |                           ^^ matched with 2 ellipses in pattern
./test/programs/error-ellipses-2-3.defs:2:9
   |
 2 |   (list $x .. .. ..))
   |         ^^ used with 3 ellipses in constructor
help: variables must be used with the same number of ellipses they were matched with
